desktop/cmp/input/CodeInput.js

```diff
@@ -215,6 +223,8 @@ export class CodeInput extends HoistInput {
             editor.setSize(width, height);
         }
 
+        const val = this.tryPrettyPrint(editor.getValue());
```

Comment:
Not 100% sure about this feature -- Doesn't it cause the control to indicate its value changed? 

Does this mean that just loading one of these things in a form will immediately dirty the form when it is shown?

Maybe we want to thing twice about monkeying with the value.

Reply:
Please remove this to a separate PR so we can merge this fix?  Thanks!

Reply:
I took it out!